tools/licenses/lib/main.dart

```diff
@@ -1235,6 +1242,33 @@ class RepositoryDirectory extends RepositoryEntry implements LicenseSource {
       result += directory.fileCount;
     return result;
   }
+
+  Iterable<RepositoryLicensedFile> get _allFiles sync* {
+    for (RepositoryLicensedFile file in _files) {
+      if (file.isIncludedInBuildProducts)
+        yield file;
+    }
+    for (RepositoryDirectory directory in _subdirectories) {
+      yield* directory._allFiles;
+    }
+  }
+
+  Stream<List<int>> _signatureStream(List files) async* {
+    for (RepositoryLicensedFile file in files) {
+      yield file.io.fullName.codeUnits;
```

Comment:
This is intended to catch renaming of files (which will affect the file paths listed in the output)